src/config.c

```diff
@@ -2686,18 +2686,11 @@ standardConfig configs[] = {
 };
 
 /*-----------------------------------------------------------------------------
- * CONFIG command entry point
+ * CONFIG HELP
  *----------------------------------------------------------------------------*/
 
-void configCommand(client *c) {
-    /* Only allow CONFIG GET while loading. */
-    if (server.loading && strcasecmp(c->argv[1]->ptr,"get")) {
-        addReplyError(c,"Only CONFIG GET is allowed during loading");
-        return;
-    }
-
-    if (c->argc == 2 && !strcasecmp(c->argv[1]->ptr,"help")) {
-        const char *help[] = {
+void configHelpCommand(client *c) {
```

Comment:
I feel like we should be opinionated and pick either having a single command and checking the argument vs having the individual sub commands have their own functions. I would prefer having the individual functions for each sub commands.

Reply:
i think we're aiming to eventually go that way (so config is an example of how it'll look), but we don't want to refactor tons of code in this PR (easy review and less damage to blame log), so the old way is still supported.

Reply:
If we agree to eventually trend there I'm okay with that, I just don't like there is code hanging around to support both.

Reply:
yes, we will "convert" all eventually

resolving convo

Reply:
Don't think we need to actively convert them.. But rather intend to phase them out gradually.